README.md

```diff
@@ -0,0 +1,49 @@
+# Phansible Role: Apache
+
+[![Build Status](https://travis-ci.org/phansible/role-apache.svg?branch=master)](https://travis-ci.org/phansible/role-apache)
+
+An Ansible Role that installs [Apache](http://httpd.apache.org/) on Ubuntu specifically created with [Phansible](http://phansible.com/) in mind.
+
+## Requirements
+
+// TODO
+
+## Role Variables
+
+// TODO
+
+## Dependencies
+
+// TODO
+
+## Example Playbook
+
+// TODO
+
+## License
```

Comment:
This is kind of legacy, it's part of my previous roles. I'd like to keep it, maybe in a slimmer way mainly because it has a nice link to the contributors list :)